sphinx_gallery/tests/test_full.py

```diff
@@ -191,7 +191,7 @@ def test_embed_links_and_styles(sphinx_app):
     assert 'pyplot.html' in lines
     assert 'matplotlib.figure.Figure.html#matplotlib.figure.Figure.tight_layout' in lines  # noqa: E501
     assert 'matplotlib.axes.Axes.plot.html#matplotlib.axes.Axes.plot' in lines
-    assert 'matplotlib_configuration_api.html#matplotlib.rcParams' in lines
+    assert 'matplotlib_configuration_api.html#matplotlib.RcParams' in lines
```

Comment:
The changes above modified how this was resolved, which I think is okay. The `rcParams` is an instance of `RcParams`, so this is still a legitimate documentation link to provide. (Whether it should choose one or the other will depend on the package itself, and we probably don't have a good way to know. Plus this should be rare, as it should be rare that packages have case-based naming differentiation in the same submodules, both of which get documented.)